File: language/dart/sdk/core/Uri.dart
void main() {
  /**
   * Static
   */
  UriParse();
  UriHttps();
}

void UriParse() {
  Uri url = Uri.parse('https://example.com/whatsit/create');
}

void UriHttps() {
  Uri url = Uri.https('example.com', 'my/endpoint/create');
}

File: language/gpl/java/jdk/java/net/URI.java
import java.net.URI;
import java.net.URISyntaxException;

class Main {
  public static void main(String[] args) {
    // Static methods
    _new();
    _create();
  }

  static void _new() {
    try {
      URI uri = new URI("https://example.com");
    } catch (URISyntaxException e) {
    }

  }

  static void _create() {
    URI uri = URI.create("https://example.com");
  }
}
